### PR TITLE
tests: update `mysql-tester` and fix some syntax problem

### PR DESCRIPTION
**Pull request by Defined2014**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/mysql-tester/pull/135

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: tests/integrationtest/r/executor/issues.result [hunk 1]
```diff
@@ -899,43 +899,43 @@ TOTAL_SPLIT_REGION	SCATTER_FINISH_RATIO
 203	1
 analyze table pt;
 set @@tidb_distsql_scan_concurrency = default;
-explain analyze select * from t order by id;  # expected distsql concurrency 2
```
Comment (tiancaiamao) [inline tests/integrationtest/r/executor/issues.result:902]: Comment is not displayed in the result file?  @Defined2014

Comment (Defined2014) [reply tests/integrationtest/r/executor/issues.result:902]: Should use `--echo xxx` to display it. Previously, comments on a single line were not displayed.

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**